src/lib.rs

```diff
@@ -6,8 +6,8 @@ pub fn cpu_usage() -> f32 {
 }
 
 /// Returns the load average in the format reported by `uptime` (1 min, 5 min, 15 min)
-pub fn load_average() -> (f32, f32, f32) {
-	todo!()
+pub fn load_average(sys: System) -> sysinfo::LoadAvg {
+	sys.load_average()
```

Comment:
Need numbers

Reply:
To clarify, you should still return an `f32` instead of sysinfo's internal arguments; we want to avoid exposing used dependencies to our own code so that we may use our mental model of how to structure our data instead of a library.

In other words, you could make a `struct` or a tuple of `f32`'s.

Reply:
LoadAvg is a _completely transparent_ struct of three `f64`s: https://docs.rs/sysinfo/0.28.2/sysinfo/struct.LoadAvg.html. We could _copy and paste_ their struct into our library, and convert the type, but that is a bad idea. 

There is absolutely no point in wrapping a library function to have it return _an identical type_ for no reason -- it would make sense if the Prometheus interface wanted 3 floats, but _we aren't writing that yet_. Duplicating library types in your own code is pointless and leads to interoperability and maintainability hell.

Reply:
This struct looks fine